oeps/architectural-decisions/oep-0003-arch-async-tasks.rst

```diff
@@ -1,6 +1,6 @@
-===================================
+###################################
 OEP-3: Asynchronous Task Management
-===================================
+###################################
```

Comment:
Ah, so does putting a row of punctuation both above and below the line introduce a new heading level?

Is this the preference for document titles? If so, should we update the RST guide?